src/BugsnagUnity/INativeClient.cs

```diff
@@ -104,5 +104,7 @@ interface INativeClient : IFeatureFlagStore
 
         bool ShouldAttemptDelivery();
 
+        void AddOnSession();
```

Comment:
Can we call it something else to avoid confusion with adding C# callbacks:

```suggestion
        void registerForOnSessionCallbacks();
```

Reply:
done 👍